### DISCUSSION TITLE
Undesired word wrap

### DISCUSSION THREAD

**Discussion by DANTHK** [gotenberg/gotenberg#1018] [Q&A]
This is source file.
[1.docx](https://github.com/user-attachments/files/17374879/1.docx)

This is output file.
[out.pdf](https://github.com/user-attachments/files/17374889/out.pdf)

I hope to get it.
[1.pdf](https://github.com/user-attachments/files/17374962/1.pdf)

The reason column wraps automatically. What should I do?
Thanks!!

**Comment by gulien** [comment]
Hey @DANTHK,

LibreOffice interprets your *docx* this way when converting it to PDF, so there is nothing Gotenberg can do there 🤔 
You can check if you can find your way with one or more form fields here: https://gotenberg.dev/docs/routes#page-properties-libreoffice

**Reply by DANTHK** [reply]
Fine....Thanks!!!
